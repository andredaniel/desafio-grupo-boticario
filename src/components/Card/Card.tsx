import React from 'react'
import { CardWrapper } from './Card.styles'

const Card: React.FC = ({ children, ...rest }): JSX.Element => {
  return <CardWrapper {...rest}>{children}</CardWrapper>
}

export { Card }
