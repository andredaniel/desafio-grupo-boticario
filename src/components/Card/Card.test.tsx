import { render } from '@testing-library/react'
import React from 'react'
import { Card } from './Card'

describe('<Card />', () => {
  it('should render Card component as expected', () => {
    const { container } = render(<Card />)
    expect(container).toMatchSnapshot()
  })
})
