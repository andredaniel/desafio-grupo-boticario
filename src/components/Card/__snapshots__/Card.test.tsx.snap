// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> should render Card component as expected 1`] = `
<div>
  <div
    class="sc-bdfBwQ fDSlMa"
  />
</div>
`;
